test/attributes.cpp

```diff
@@ -256,3 +256,136 @@ TEST(attributes, user_op)
     EXPECT_EQ(g_oracle->get_hyper_parameters(), oracle->get_hyper_parameters());
     EXPECT_EQ(g_oracle->get_ultra_parameters(), oracle->get_ultra_parameters());
 }
+
+TEST(attributes, shuffle_channels_op)
+{
+    FactoryRegistry<Node>::get().register_factory<opset1::ShuffleChannels>();
+    auto data = make_shared<op::Parameter>(element::i32, Shape{200});
+    auto axis = 0;
+    auto groups = 2;
+    auto shuffle_channels = make_shared<opset1::ShuffleChannels>(data, axis, groups);
+    NodeBuilder builder(shuffle_channels);
+    auto g_shuffle_channels = as_type_ptr<opset1::ShuffleChannels>(builder.create());
+
+    EXPECT_EQ(g_shuffle_channels->get_axis(), shuffle_channels->get_axis());
```

Comment:
@diyessi @tsocha Can we add several tests which create operation with using visitors but not all attributes are defined?

For example (for this case) what happened if axis is not defined? Can we detect that axis wasn't defined?

These tests can be implemented in the separate PR.